docs/hooks/hook_civicrm_alterReportVar.md

```diff
@@ -20,6 +20,14 @@ This hook is used to add or modify display columns and filters.
 
 -   null
 
+markdown_extensions:
```

Comment:
This will be printed in the content - we already handle enabling the extensions at the book level It's just the syntax that needs fixing.